docs/_static/css/custom.css

```diff
@@ -9,3 +9,15 @@
 #rtd-stickybox-container {
   text-align: center;
 }
+
+.wy-menu.wy-menu-vertical a.reference.external::after {
```

Comment:
I'd argue for this on _all_ external links, but baby steps :)

Though [@humitos had a good point](https://github.com/readthedocs/readthedocs.org/pull/10026#issuecomment-1432624078) that it makes the link more visible, but I think it's good to be explicit here?

Reply:
![image](https://user-images.githubusercontent.com/374612/219450699-95e3ec71-690a-4e9b-9fc8-40e9c5d265c6.png)

> Though https://github.com/readthedocs/readthedocs.org/pull/10026#issuecomment-1432624078 that it makes the link more visible, but I think it's good to be explicit here?

The font-awesome icon is much more discreet so I think that it works better.

Reply:
I get a feeling that this will cause unintended side-effects.. like if the `<a>` is on an `<img>` tag

Reply:
Yea, we likely need to scope it smartly. I'm sure there's good examples of doing this other places, but we can start w/ the toctree for now.